### PR TITLE
WWW-384: Fix responsiveness on Edge 42/44 

### PR DESCRIPTION
**Pull request by adamszalapski**
## Jira

https://pegadigitalit.atlassian.net/browse/WWW-384

## Summary

Create Edge ~v.42 specific CSS to handle the video player "play" icon

## Details

Make play icon responsive

## How to test

View "/pattern-lab/?p=components-video" in Edge v.18 Browserstack (this browser also doesn't support clip-path)
Check it on different screen sizes.

### CODE REVIEW

**Review by colbytcook — APPROVED** [review]

File: packages/components/bolt-video/src/_videojs-enhancements.scss [hunk 1]
```diff
@@ -87,6 +84,23 @@ $bolt-video-js-button-shadow-level-hover: 'level-40';
...
+
+        @include bolt-mq(xxlarge) {
+          font-size: 100%;
+        }
+
+        &:before {
+          position: absolute;
+          top: 0;
+          left: 0;
+          transform: translate(-65%, -30%);
```
Comment (danielamorse) [inline packages/components/bolt-video/src/_videojs-enhancements.scss:100]: @adamszalapski can we not use `translate(-50%, -50%)` here and use top/left `50%`?

**Review by danielamorse — APPROVED** [review]
I can't really test this because the `bolt-ms-edge-42-only` mixin doesn't appear to apply to Edge 18, which is the closest thing I have to 42. So, if @colbytcook reviewed and approved, I'm good with it.

@adamszalapski had just one inline comment/question.

File: packages/components/bolt-video/src/_videojs-enhancements.scss (hunk 1 above)
Comment (adamszalapski) [reply packages/components/bolt-video/src/_videojs-enhancements.scss:100]: @danielamorse ok i can try to figure out the better numbers for top and left 👍

Comment (colbytcook) [reply packages/components/bolt-video/src/_videojs-enhancements.scss:100]: @danielamorse I have confirmed that "bolt-ms-edge-42-only" does work (remember 42 = 17 in Edge terms).

Comment (danielamorse) [reply packages/components/bolt-video/src/_videojs-enhancements.scss:100]: ah! Good call. Thanks for sharing.

Comment (adamszalapski) [reply packages/components/bolt-video/src/_videojs-enhancements.scss]: Done

**Pull request merged**